src/app/Models/Traits/HasEnumFields.php

```diff
@@ -20,9 +20,13 @@ public static function getPossibleEnumValues($field_name)
 
         $instance = new static(); // create an instance of the model to be able to get the table name
         $connectionName = $instance->getConnectionName();
-
+        $connection = DB::connection($connectionName);
```

Comment:
```suggestion
        $connection = $instance->getConnection();
```
Small suggestion to make things slightly more simple ;)

Reply:
I was not able to merge your suggestions @jnoordsij 

![image](https://user-images.githubusercontent.com/7188159/219399424-7d89577e-21ae-4095-876a-ffcf0b499930.png)

but I agree with it, thanks for pointing out. I've commit it directly.